kvm/requests.yaml

```diff
@@ -15,6 +15,9 @@ releases:
         - name: app-operator
           version: ">= 3.2.1 < 4.0.0"
           issue: https://github.com/giantswarm/giantswarm/issues/15920
+        - name: chart-operator
```

Comment:
After we had made a performance fix, giantswarm/giantswarm#16406, we released v4.2.0.